development/setup.js

```diff
@@ -3,29 +3,31 @@ const waitFor = require('./utils/wait-for.js');
 const runCommand = require('./utils/run-command.js');
 const ensureDotEnv = require('./utils/ensure-dot-env.js');
 const dnsIsOk = require('./utils/check-if-dns-is-correct.js');
+const createCypressEnvFile = require('./utils/create-cypress-env-file.js');
 const path = require('path');
 const fs = require('fs');
+const os = require('os');
 
 const startingDockerCompose = () => runCommand('docker compose up -d --remove-orphans');
 
-const createAdminUser = (env) =>
+const createUser = (username, password, admin) =>
   runCommand(
     [
-      `docker exec studio-repos gitea admin user create`,
-      `--username ${env.GITEA_ADMIN_USER}`,
-      `--password ${env.GITEA_ADMIN_PASS}`,
-      `--email ${env.GITEA_ADMIN_USER}@digdir.no`,
-      `--admin`,
+      `docker exec studio-repositories gitea admin user create`,
+      `--username ${username}`,
+      `--password ${password}`,
+      `--email ${username}@digdir.no`,
```

Comment:
Just an idea... but in light of this being open source and can potentially be used by many... we should at some point consider taking the domains out to the .env as well...

Reply:
Yes, that would be a very good idea. But I think that is an separate issue.